DESIGN.md

```diff
@@ -0,0 +1,40 @@
+# Design
```

Comment:
If this file should be a guide to contributors we may want a paragraph on our themes, how attributes should be used and probably link to that video I always link to, a section pointing to our typography and the typography system of material in general would also be worthwhile I think

Reply:
Also Night Mode is a must mention as any UI change should work in night mode

Reply:
Nice suggestions, will add.

Reply:
There isn't a typography section, granted our typography is not excellent in the app, mostly link/paraphrase this https://material.io/design/typography/the-type-system.html#type-scale
link to our type.xml file and say that to use it you reference by the `textAppearance*` attrs